driver-redpanda/src/main/java/io/openmessaging/benchmark/driver/redpanda/RedpandaBenchmarkConsumer.java

```diff
@@ -68,13 +79,14 @@ public RedpandaBenchmarkConsumer(KafkaConsumer<String, byte[]> consumer, Consume
                     long now = System.nanoTime();
                     long timeSinceOffsetCommit = now - lastOffsetNanos;
                     if (!offsetMap.isEmpty() && timeSinceOffsetCommit > TimeUnit.SECONDS.toNanos(5)) {
-                        log.info("msec since last offset commit: {}", (now - lastOffsetNanos) / 1000 / 1000);
+                        log.debug("msec since last offset commit: {}", (now - lastOffsetNanos) / 1000 / 1000);
```

Comment:
Thanks for this, I had not intended to leave this very verbose line at `info`. Arguably it should be moved out of here entirely.